cypress/integration/users/delete-users.spec.js

```diff
@@ -0,0 +1,72 @@
+import {
+  Button,
+  including,
+  Pane,
+} from '../../../interactors';
+
+describe('Deleting user', () => {
+  const lastName = 'Test123' + Number(new Date()).toString();
+  const ResultsPane = Pane({ index: 2 });
+  const ActionsMenu = ResultsPane.find(Button({ className: including('actionMenuToggle') }));
```

Comment:
what about new interactor for ActionsMenu?

Reply:
I'm afraid I'm not so experienced with interactors yet

Reply:
@poflankov have a look here. https://github.com/folio-org/stripes-testing/blob/master/doc/interactors.md 


Also, for a general introduction to interactors https://frontside.com/interactors/docs

Reply:
I tried to make the ActionMenu not a separate interactor, but the Pane interactor method.
Please take a look.